docs/user/dql/functions.rst

```diff
@@ -1747,21 +1747,21 @@ Description
 >>>>>>>>>>>
 
 Usage: minute(time) returns the minute for time, in the range 0 to 59.
+The `minute_of_hour` function is provided as an alias.
 
 Argument type: STRING/TIME/DATETIME/TIMESTAMP
 
 Return type: INTEGER
 
 Example::
 
-    os> SELECT MINUTE((TIME '01:02:03'))
+    os> SELECT MINUTE((TIME '01:02:03')), MINUTE_OF_HOUR((TIME '01:02:03'))
```

Comment:
Does `TIME '01:02:03'` work? I think it should be `TIME('01:02:03')` or just `'01:02:03'`.

Reply:
Fixed in 6deaafb541257764eae4ec96409144b04ba455bd